docs/versioned_docs/v0.8/basics/components.md

```diff
@@ -12,6 +12,7 @@ In order for the `view!` macro to distinguish between regular elements and compo
 convention to name components using `PascalCase`.
 
 ```rust
+#[allow(non_snake_case)]
```

Comment:
This is already added by the `#[component]` macro. The IDE might not have expanded the macro (in certain cases with intellij rust or old versions of rust-analyzer) and therefore shows the issue.

Reply:
intellij rust does not show this indeed. Thats why I added it because it gives warnings. Removed it.